### DISCUSSION TITLE
Possible migration issue with current rc? System carrier moved from Fedex to eShipper

### DISCUSSION THREAD

**Discussion by ChrisNolan** [karrioapi/karrio#658] [General]
I changed my docker-compose.yml to use the new rc32 images... I brought it down, and then up again... all looked fine.

But then I was getting "No active carrier connection found to process the request" as a response to my API calls... looked at :3000/connections/system and I saw my 'fedex-ca-tracking' carrier-id still there so... oh wait, but teh big blue label next to that wasn't saying "Fedex" anymore, it was saying "Eshipper"?  Huh?  Checked :5002/admin/providers/eshipperconnection/ and sure enough, there was my fedex-ca-tracking instead of having it over at :5002/admin/providers/fedexconnection/

Any ideas what could have done that?  Was it a bad setup on my end, or an issue with one of the 'migrations' or something?

Let me know if I can check any logs or anything.

In the mean time I'm just going to re-setup my tracker under Fedex (I don't see an option to 'move' it or anything).

Thanks

**Comment by ChrisNolan** [comment]
~~Huh... now fedex's dev portal is having problems and not responding.  just my luck ;-)  Any hints on how I can 'move' that one back over to fedex then in the database so I can still use my api secret attached to it?  (of course I didn't save my secret anywhere else)~~

Update: nvm dev portal back up and running so I've gotten a new secret after all.  Got to explore a few parts of the system I hadn't seen yet though... got confused when my 'eshipper-connections' table was empty... I think the 'system' ones must be somewhere else... found the graphQL for it in the GET_SYSTEM_CONNECTIONS const but that was as far as I got on that journey.

**Reply by danh91** [reply]
Hi @ChrisNolan , 

🤔 That would definitely be a migration issue.

What version did you have before?


Two ways to fix the FedEx -> eshipper issue. 
- connecting to your database and changing the `carrier_code` field to from `eshipper` back to `FedEx`

- ssh to your docker container with docker run and use karrio shell. 

I will share the commands from my pc

**Reply by ChrisNolan** [reply]
I was on... rc23 before I believe, might have been 24.  I dont need the commands at this point, I've gotten it moving again by re-creating my system carrier as I only had a handful in there.

**Comment by zaryabanwar** [comment]
i have almost the same issue.
when we try to call api to get rates it returns a response "no active carrier connections exist".
the carrier exist in the db carrier_providers table but not in the carrier-config table.
also not able to load carrier list in dashboard and all graphql network requests going to api.karrio.io/graphql

**Reply by danh91** [reply]
HI @zaryabanwar ,
I don't know if you have the same issue as described above.

- Can you confirm whether you are in test or live mode? 
- Also, if you are using API requests, can you confirm whether the API key is the one from the test or the live move?
- What is the previous version of Karrio you were migrating from?

**Reply by zaryabanwar** [reply]
Hi @danh91 

- We have deployed on digital ocean following the karrio documentation using docker pre-configured image and we are in test mode but the environment is set to production mode for karrio as we can set carrier test-mode from admin.
- The API keys for carriers are for test environment and we sent test_mode in payload.
- we deployed new with KARRIO TAG LATEST
